<!DOCTYPE html>
<html>

<head>
  <meta charset="utf-8" />
  <title>CooVar: Co-occurring variant analyzer</title>
  <meta name="keywords" content="Technical Note" />
</head>

<body prefix="sa: https://registry.standardanalytics.io/ saterms: https://standardanalytics.io/ns/ schema: http://schema.org/ deo: http://purl.org/spar/deo/ sro: http://salt.semanticauthoring.org/ontologies/sro# bibo: http://purl.org/ontology/bibo/ doco: http://purl.org/spar/doco/ xsd: http://www.w3.org/2001/XMLSchema#"
resource="sa:bmc-res-notes-vergara-2012" typeof="schema:MedicalScholarlyArticle">
  <meta property="schema:keywords" content="Technical Note" />
  <meta property="schema:version" content="0.0.0" />
  <article>
    <section typeof="doco:FrontMatter">
      <h1 property="schema:headline" about="sa:bmc-res-notes-vergara-2012">CooVar: Co-occurring variant analyzer</h1><span property="schema:datePublished" content="2012-10-31T19:00:00.000Z" about="sa:bmc-res-notes-vergara-2012">Wed Oct 31 2012</span>
      <ul>
        <li rel="schema:author" about="sa:bmc-res-notes-vergara-2012">
          <div typeof="schema:Person"><a property="schema:email" href="mailto:ivergara@gmail.com"><span property="schema:givenName">Ismael A</span>  <span property="schema:familyName">Vergara</span> </a>
            <ul rel="schema:affiliation">
              <li typeof="schema:Organization"><span property="schema:description">Department of Molecular Biology and Biochemistry, Simon Fraser University, 8888 University Drive, Burnaby, B.C., V5A 1S6, Canada</span>
              </li>
              <li typeof="schema:Organization"><span property="schema:description">GenomeDx Biosciences Inc, 1595 West 3rd Avenue, Vancouver, BC, V6J 1J8, Canada</span>
              </li>
            </ul>
          </div>
        </li>
        <li rel="schema:contributor" about="sa:bmc-res-notes-vergara-2012">
          <div typeof="schema:Person"><a property="schema:email" href="mailto:frech.christian@gmail.com"><span property="schema:givenName">Christian</span>  <span property="schema:familyName">Frech</span> </a>
            <ul rel="schema:affiliation">
              <li typeof="schema:Organization"><span property="schema:description">Department of Molecular Biology and Biochemistry, Simon Fraser University, 8888 University Drive, Burnaby, B.C., V5A 1S6, Canada</span>
              </li>
            </ul>
          </div>
        </li>
        <li rel="schema:contributor" about="sa:bmc-res-notes-vergara-2012">
          <div typeof="schema:Person"><a property="schema:email" href="mailto:chenn@sfu.ca"><span property="schema:givenName">Nansheng</span>  <span property="schema:familyName">Chen</span> </a>
            <ul rel="schema:affiliation">
              <li typeof="schema:Organization"><span property="schema:description">Department of Molecular Biology and Biochemistry, Simon Fraser University, 8888 University Drive, Burnaby, B.C., V5A 1S6, Canada</span>
              </li>
            </ul>
          </div>
        </li>
      </ul>Accountable Person:
      <ul>
        <li rel="schema:accountablePerson" about="sa:bmc-res-notes-vergara-2012">
          <div typeof="schema:Person"><a property="schema:email" href="mailto:chenn@sfu.ca"><span property="schema:givenName">Nansheng</span>  <span property="schema:familyName">Chen</span> </a>
            <ul rel="schema:affiliation">
              <li typeof="schema:Organization"><span property="schema:description">Department of Molecular Biology and Biochemistry, Simon Fraser University, 8888 University Drive, Burnaby, B.C., V5A 1S6, Canada</span>
              </li>
            </ul>
          </div>
        </li>
      </ul>
      <section>
        <p>Copyright ©2012 Vergara et al.; licensee BioMed Central Ltd.</p>
        <meta about="sa:bmc-res-notes-vergara-2012" property="schema:copyrightYear" content="2012" />
        <div about="sa:bmc-res-notes-vergara-2012" rel="schema:copyrightHolder">
          <meta typeof="schema:Organization" property="schema:name" content="Vergara et al.; licensee BioMed Central Ltd." />
        </div>
        <section about="sa:bmc-res-notes-vergara-2012" property="schema:license" content="http://creativecommons.org/licenses/by/2.0">
          <p>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/2.0" class="uri">http://creativecommons.org/licenses/by/2.0</a>), which permits unrestricted
            use, distribution, and reproduction in any medium, provided the original work is properly cited.</p>
        </section>
      </section>
      <section rel="schema:abstract" about="sa:bmc-res-notes-vergara-2012">
        <div typeof="schema:Abstract">
          <section typeof="schema:Abstract" property="schema:hasPart">
            <h3 property="schema:headline">Background</h3>
            <p property="schema:abstractBody">Evaluating the impact of genomic variations (GV) on protein-coding transcripts is an important step in identifying variants of functional significance. Currently available programs for variant annotation depend on external databases or annotate
              multiple variants affecting the same transcript independently, which limits program use to organisms available in these databases or results in potentially incorrect or incomplete annotations.</p>
          </section>
          <section typeof="schema:Abstract" property="schema:hasPart">
            <h3 property="schema:headline">Findings</h3>
            <p property="schema:abstractBody">We have developed CooVar (Co-occurring Variant Analyzer), a database-independent program for assessing the impact of GVs on protein-coding transcripts. CooVar takes GVs, reference genome sequence, and protein-coding exons as input and provides
              annotated GVs and transcripts as output. Other than similar programs, CooVar considers the combined impact of all GVs affecting the same transcript, generating biologically more accurate annotations. CooVar is operated from the command-line
              and supports standard file formats VCF, GFF/GTF, and GVF, which makes it easy to integrate into existing computational pipelines. We have extensively tested CooVar on worm and human data sets and demonstrate that it generates correct annotations
              in only a short amount of time.</p>
          </section>
          <section typeof="schema:Abstract" property="schema:hasPart">
            <h3 property="schema:headline">Conclusions</h3>
            <p property="schema:abstractBody">CooVar is an easy-to-use and lightweight variant annotation tool that considers the combined impact of GVs on protein-coding transcripts. CooVar is freely available at <a href="http://genome.sfu.ca/projects/coovar/" class="uri">http://genome.sfu.ca/projects/coovar/</a>.</p>
          </section>
        </div>
      </section>
    </section>
    <section property="schema:articleBody" about="sa:bmc-res-notes-vergara-2012">
      <section>
        <h2>Findings</h2>
        <section typeof="deo:Introduction">
          <h3>Introduction</h3>
          <p>One central goal of many genomics projects is to detect different types of genomic variations (GVs) and to understand how these GVs explain differences at the phenotypic level, for example, between healthy and diseased individuals [<a href="#B1"
            class="bibr">1</a>,<a href="#B2" class="bibr">2</a>]. Accurate and comprehensive detection of GVs, including single-nucleotide variations (SNVs), insertions and deletions, has been greatly facilitated by the development of next generation
            sequencing technologies [<a href="#B3" class="bibr">3</a>] and variation detection methods [<a href="#B4" class="bibr">4</a>]. After GVs are defined, evaluation of their functional impact on protein-coding transcripts becomes the primary focus.
            Many programs have been developed for this task, of which Ensembl’s Variant Effect Predictor (VEP) [<a href="#B5" class="bibr">5</a>], GATK’s VariantAnnotator [<a href="#B6" class="bibr">6</a>], Sequence Variant Analyzer (SVA) [<a href="#B7"
            class="bibr">7</a>] and ANNOVAR [<a href="#B8" class="bibr">8</a>] are among the more popular ones.</p>
          <p>Current variant annotation programs have important limitations. First, they assess the effects of multiple co-occurring GVs on the same transcript independently, which can be problematic when nearby GVs alter each other’s effect [<a href="#B9"
            class="bibr">9</a>]. For example, a small deletion can restore the open reading frame (ORF) disrupted by a small insertion co-occurring nearby on the same transcript. A second limitation is that most programs are tightly coupled to external
            databases, making their use inconvenient or even impractical for users who work on organisms whose genome sequence or annotation is not available in these databases.</p>
        </section>
        <section>
          <h3>Implementation</h3>
          <p>We have developed an easy-to-use Perl program named CooVar (Co-occurring Variant Analyzer) to address these limitations. CooVar takes as input (i) a list of GVs in the popular Variant Call Format (VCF) [<a href="#B10" class="bibr">10</a>] or
            in a simpler tab-delimited file format, (ii) the reference genomic DNA sequence in FASTA format, and (iii) protein-coding exon coordinates in GFF or GTF format.</p>
          <p>The core output of CooVar are two files: a GVF file [<a href="#B11" class="bibr">11</a>] reporting the functional impact of each input GV on transcripts, and a GFF file including (i) the transcript models, (ii) all GVs impacting each transcript,
            and (iii) a prediction of how GVs impact the function of each transcript. The functional impact of GVs on protein-coding transcripts is annotated as: ORF_INTACT, if the transcript is not impacted by any GVs; ORF_PRESERVED, if the transcript
            is impacted by GVs but these GVs do not introduce internal stop or splice site variants; ORF_DISRUPTED, if an internal stop or splice site variant is present; and FULLY_DELETED, if the transcript is deleted. In the case of a transcript that
            has its ORF disrupted by an internal stop codon, CooVar provides the percentage location of the first internal stop codon in the variant peptide compared to the reference.</p>
          <p>CooVar classifies GVs according to the GVF v1.05 specification for structural variants described in the Sequence Ontology (SO) Project [<a href="#B12" class="bibr">12</a>]. For SNVs, these categories include <em>silent_mutation</em>, <em>synonymous_codon</em>, <em>conservative_missense_codon</em>, <em>non_conservative_missense_codon</em>, <em>stop_gained</em>, <em>stop_lost</em>, <em>splice_acceptor_variant</em>,
            and <em>splice_donor_variant</em>. Insertions and deletions are classified into the categories <em>silent_mutation</em>, <em>frameshift_variant</em>, <em>inframe_variant</em>, <em>splice_acceptor_variant</em>, and <em>splice_donor_variant</em>.
            The functional impact of missense SNVs causing amino acid changes is further evaluated with the Grantham score [<a href="#B13" class="bibr">13</a>] and annotated as <em>CONSERVATIVE</em> or <em>MODERATELY_CONSERVATIVE</em> (both classified as <em>conservative_missense_codon</em>)
            and <em>MODERATELY_RADICAL</em> or <em>RADICAL</em> (both classified as <em>non_conservative_missense_codon</em>) [<a href="#B14" class="bibr">14</a>]. For SNVs impacting protein-coding exons, CooVar also reports both the amino acid change and
            the codon change between the reference genome and the variant. This allows the user to observe immediately if a change in a codon is caused by one, two or three co-occurring substitutions at the same codon. Furthermore, CooVar lists separately
            all those SNVs that fall into multiple categories by impacting two or more protein-coding transcripts differently (e.g. synonymous <em>vs</em>. missense).</p>
          <p>In addition to the annotation of individual transcripts and GVs, CooVar outputs various summary statistics. For example, CooVar generates statistics on the codon bias for synonymous versus non-synonymous SNVs. In two other files CooVar outputs
            the length distribution of indels across the whole genome versus the length distribution of indels impacting only protein-coding transcripts as a way to detect biases towards non-frameshift indels in exonic regions. The file <em>variant.stat</em> provides
            information on the distribution of internal stop codons and on the total number of transcripts affected by SNVs, insertions and deletions, or by any combination of those. If the --<em>circos</em> flag is used, CooVar computes the genomic distribution
            of SNVs, insertions, deletions and coding exons in a format compatible with the Circos tool for visualization [<a href="#B15" class="bibr">15</a>].</p>
          <p>One advantage of CooVar over other programs is that it provides full-length variant transcript and protein sequences in FASTA format as output, which can be useful for downstream analyses (for example for sequence alignments). The same information
            is provided at the exon level in two additional files. Since a direct comparison between the reference and variant transcript is also desirable, CooVar provides an exon-based alignment of reference and variant sequences for each transcript,
            with variant nucleotides marked in uppercase. This makes it easy to spot all SNVs, insertions and deletions that impact a given protein-coding transcript in a region of interest.</p>
          <p>Another commonly requested feature in variant annotation is to identify GVs that overlap with protein domains. This is because GVs affecting conserved domains are more likely to be of functional importance. With CooVar this analysis can be performed
            in two steps. First, the script <em>protein2genome.pl</em> can be used to map protein (domain) coordinates to the genome, which generates a GFF file with genomic coordinates. The script <em>annotate-regions.pl</em> can then be used to compute
            the overlap between this GFF file and the CooVar GVF output file. Overlap computation is performed efficiently using interval trees and generally finishes within a few minutes, even for very large data sets. The result of this two-step process
            is a new GVF file in which GVs are annotated with the protein domains they overlap with. It is worth mentioning that <em>annotate-regions.pl</em> script is generic and can also be used to annotate GVs that overlap with non-protein-coding regions
            (for example transcription factor binding sites) as long as coordinates for these regions are provided in the required input GFF format.</p>
          <p>More detailed information about program parameters and input file formats can be found in the program README file or in the Perl scripts themselves.</p>
        </section>
        <section typeof="deo:Results sro:Discussion">
          <h3>Results and discussion</h3>
          <p>We have tested CooVar on two datasets, both of which are available from the project website. The first dataset corresponds to 120,638 GVs (116,999 SNVs, 1,553 insertions ranging from 1 to 34 bp in length and 2,086 deletions ranging from 1 to
            24 bp in length) detected in the Hawaiian isolate CB4856 of the model organism <em>Caenorhabditis elegans</em>. CB4856 GVs, the N2 reference genome (isolated in Bristol, England) and 24,256 annotated N2 protein-coding transcript models were
            obtained from WormBase release WS210 [<a href="#B16" class="bibr">16</a>]. CooVar had a processing time of 10 minutes for this data set and classified 15,293 transcripts as ORF_INTACT, 8,446 transcripts as ORF_PRESERVED, and 517 transcripts
            as ORF_DISRUPTED. Figure <a href="#F1" class="fig">1</a> shows a Circos image with the distribution of SNVs, deletions, and insertions along the six <em>C. elegans</em> chromosomes. This image was generated by using the CooVar output files (option
            --<em>circos</em>) as input for Circos (version 0.62-1) [<a href="#B15" class="bibr">15</a>].</p>
          <figure about="sa:bmc-res-notes-vergara-2012" rel="schema:hasPart" id="F1">
            <img about="sa:bmc-res-notes-vergara-2012/F1" src="data:image/gif;base64,R0lGODlhZABjAPcAABoaGncAAHsAAEREAExMAFBQAF5eAltbEnNMAGZmAGRkCmZmFGxsHW1tFW5uIHJyJHR0Knp6M319OkREREtLS1VVVVtbW2RkZG1tbXR0dHt7e4cAAJUAAKYAAK4AALQAALsDA7gMDLwUFLodHbwgIL8uLr4zM8UCAswAAMQNDdAAAMMTE+MAAO0EBOsJCfQCAv4AAPEICOkXF+waGuEZGcMpKcA2NsI6Ot43N+MjI+ogIOMqKuI7O+E0NIJXAIZaAI1fApRjAplmAJZnCJdpDpBnFJlvHJtxHJVvI5dwJJt0JJdyKZ53KZ54Kpx6NaJ+N71CQr9zc8NFRcNISN9HR8VTU8hcXN9WVuJKSuJDQ+NeXspqaspkZN9sbN1iYst0dMx7e99zc997e9N2duNmZuJpaeJ1deJ7e4KCGo2NHJaWHpmZHpWVIpaWLJmZL5aWNJqaNJeXOpqaO6SBPYODQ4aGSYmJTZ6eSJubQY6OVaKCQqWGSaeJTauOVK2SXKCgRqCgTKmpXaOjVpaWY5uba66UYbGXZLSdb6ioYa6ubaWlfqKid7ejebijebGxdLKyeoWFhYuLi5WVlZqamqurhq6ujb+ui7a2g7a2jLW1lb+/lLmwnbe3m6Ojo6ysrL6+p7y8qrS0tLy8vMyDg86NjdGBgd+AgNiLi82dndWXl9KZmd+YluKFheOLi+OVleKfn8KzlMa4nuWjmceiot2urtOjo8W5osm/v8O0tNWwsNmwsNy/v+WkpOerq+iqquu1rOywo/C7r+K6uumzs8DAmcbGpcTErM3CrsrKrcbGsc3NscrKvdDHt9LKudXVv/rIuf3Rv8XFxc3MwszMzNbMzNTTw9XUytrWzdnZzNnZxtTU09vb1Nzc3NvV1ebCwujAwOPNzevNzf3Uxf7Xyf7bzePU1OrW1uTc2+vc3OHc0/7e0eLh1uPi2+jk3v7h1f7m2v7o3uPj4+zj4+rq5Ofo6Ovr6/7s5PPs7P7v6PHw7v/y7PT09P/28v/59vf4+P///yH5BAAAAAAALAAAAABkAGMAAAj+AP8JHEiwoMGDCAfGq4eQW7yEECNKnEixIsF43Az6+xdv376NFkOKHGlx2jRRniZJkhSp2zUgMIss0VMIFjN6A/1FI8mzJ8J40TqtnESUqKRJ2q4JWSokCNOmSQrFusfQp1WS8UKpLMqVa9IhT8MyDfIEVr6raA96zDltq9GuRVlOu/YDplOxS+8O8dMsrd9/3LT92yfqKFyikSaFiqZN1DRuIAfPs3bMkp8ld/E6OfYXLVDDXY+K6rax47TI/7RFixev30OB3I4dSpKZ6Q8nfTvz1HZY9GPW2n4b/Ag8WjRu0/b9XsYoCVMEP5bqOScwnmDdEukJfatyscNo+/7+7UQN0R/5fQ438Qni42kQRv46Yo84TVIGDHFFsQ7PUSB5nvFZ40dtQiBxzXwJ7ZPSJJFQEIkkntDDGoKw3VPNHmENYQmFBWkHWmLcOIQThxx1Q88xRtTmx3+6cROaKPVkZJE/Esaj0zQLhYTePn6E9cQ8FHYDF0avReTPSZFEAskk0+DUSSjidVNBSkWWh6MtRDylBHWd7aQNaBDWGNE+oWTQpCj+iMIfbDJyw9hCnXiCZnmApdPEU0eso1s0Q6mkX5UHcSMKBdf1t+aY/3jiSTQjNtRNPnpkxkQ7f3ED5nGAFhTNYvUk5xNO3IRS1UHj9ciUEqNaFQ9XkjzWqEH+2kWyUTd+RRJPJKkOFOJDhjw1x0bc5Kojq9P0d5A/3XTyz6sF8aMPP/b0w08/5Lxjjz7Y9gORBZ3IeBGR+/TxlB/osUiRP51wpSaOgaLJzaEC9aOPPfDA8868zlJrLb334nuQYPTMWRA33bDjjx5MEbENYDyJ4lYo+9B6kMOZ/iPvvO/U68682P5TrT30woMtyNdqaxByEl+EXD5MCMEEO4DRY25ElhbViTY2ksoNs/9Aa8870GacMb766GsPPkPDQ68+8MhrMkH+aCDKQaeds+I+g0AA77ldSYgQlAZJS7LI/A5dtLbYYisy0/ReO+/RTws0DQBgF8QOkNg8kED+AouM5HBRj3lLkMAEjTyv0msXLZG818Ljs732FERjJ8qqNcjeey9sET2geaLcQZMILtA76rhNsj4jyevzO/fyw49BnexkUEd67y1BSAuq1OlB9YRS7ED90Pux2z7Jyzo+jD9L0D4YiP7P1MZgnsAnE9HjEGirTZPrYwUFDzI85IicVvDOXvvMO3FbahC7dmD+wMw5HckVPeEduk8Fv8f7s9JLdxa8Ps8IHzw0Mrt/sEMBmKOeRO4ht0kcZ2sZiEcn1vSs/UULO/0QR/je4Q53FKQeniAIPbSxEUJgjgHwE0gXcuCKjZjHIKKQ3UCcNbILzocf9lKHOsRHkG4NhB7+TfrHPKSnwIlx4x4ugAEMZICOY03iUD9L27Q4xA9y7DBacYuHJDS1kcvtrQ4J8cc+SqFEGOwAMOSZhgwtZq/WxQ1BOIzcvNKXP450xBqYM0DFBtKDMpriSNxLCMlY90YKvQ4f8PjFL6CmAahB5h8RwJwijFUQc7xAiS+Qxz+60aSqMOttIEMdiQhiD3KsIhjvAEk+KnCB/NUDKP/IBOYgEKODtKKMPBBIyv7hDwxUjo38i9woCcIPWZROH/hIFCS40YlISI4d2kBg5hCCgzK6YjB1NMmhHlfIUWILH+4ghyh52QlARawOmKME71qASXQcSXAP4k8/RCatcQ6TIO/+qJY6yEGQUIRwYJXAHB0OIowy5uAfnYpMPbSRssPd654HwQc5xCEOdbzOP3VTCB73poD/hKGMZVhWlTxxAYLUi38Q1Qg53BFKlRmEHQvAXDUMQoUyvkI5jxQIJDLwRItxkIMXTWnhvidMgZCpIBihA+YqYRAalBEcnyMIBkwyNX3sUx3vEKpB4ogPxQ3kl7pShAEGUIBBFGQf7IRBCzZCwh+usV4/y6pWC+JQe2YUNskYKwFuR5ByGBQ2oZBYNDJASv7Zc67/sIc7WFdU8cCLG9I4YUFoUcYs+Cd24ZlEkfqRMR12U6iKdYc6PJgT52nDAHszAEhs5IoykoEeoYj+RzeqoibghcyGiBUIPzIGspwcZ3Yx3ZuMusGNMZTRDCYqyD8FYg9xjGO0uSVIP/BBXRvqZJeDiUftElCof3xUiWdwniTWxA98fC+6wGMsbt1kkG5IAHPLIIgZysgK5zVyIPpgnbXQCzxpTXEgd+XIe/cGCnbIrgv0JZxApopfkjU2ummzJyclx40BJ0AaHxHId2HAikxNbYYg6yp/LaZDcpAjqFo8Kz0sHN+BzFeJHTZIBEFMshHPs41BbSaFI7m3FgtEDMe1TqogQUwH2zhj9TIZn+7aEQdgTnMCaa0StSAh1ITOtvz7bEr1sdJ9EmQSy0VPRoKbgCpRVolY8HD+ZLynPP4yzcSkHUwkssnLPB4KHGXcQaa0cah62iOoueVXb6szgVTFIx/bwNwCOnRJGLggZyJcIygfPNdxONeiAKYAUuMRPds1FQYveEE5MuVM/FrLq9GFFjzUgY+TZOTDuvoHJTBn1oJg4QMdgMEqnAeJUsfLYq5L9UWDyo1ILJeXO8nDUg0iBhV8gAVkyNSTihzOASK2H7/YIT5QQ1yXMgBz1rgIPVCR5xyJsEqltFbTEAutXwBDHPBzCDfsTBCHoBWT1PgPk7AcTkrfM8T4eMZF1sgRf3BCoAjhQRlHsTU5rWnVvAX0PfUhWvPGjXv7KJI/MuHkBCjihQUxRRn+0+wf5NTDHxlY0zxPJ9TgWQurBMmAr+NTJWsoYhvsrWSjX0ANjIQisAKZM0HMW8OUls9eQaXHBD68D8EcqRhxwATMINJHJbYiTYKrB6x1i0iKalk3rktbUXcyiUgLRhBrWIMaKsGOPd5SiWd0npyIOVFy/PnrfmFzvnKyKILkYxr5YIca1K4GdmT4IPKIQRmFMY3ZEqQeGCiIFZ/FOhJ5L5RvvK9CMuKIwasBEBK5Ai5fub461vOn1p4P40iWTIJ4YpeE6cY82JD2NShDIuFoNAy8EQ+e9bogHVz1ujHYthov7670oNUjai+HFAoEC7jkSDRQsw8NbHPVf8ahxK/+AkqUFmSLSK3H7GtPDIp4Q/e8yGlkuoGBbWJRaeIcnz2uaPyB8IZnD0lE7ePg/IFAX4kycA49hzOu91vAk1jzd1vFw2X1Ej7URR6e0F0DMQ3OMHhphwwWgQ5pBQMh9Q+eEnMaAGkWQy+lAw8d9GckkV+LtUHu0Hr9tEfcIAe1dwf9Bxvx8GKg9g2AdChRQ3D/IDINyD9d1Q94ZzFE+A8U5w7gYy+HhU0/sXxpxwbYMCNipANlpAMM1BGpgnVbNX/74jjX4g5EOC2uM4bzEk4dw1nVJoYdohoNkQwWuAaPQBLfoHtYsFDTpxGHN3QOxjFieHRMmF8mKA7XwlyL1U3+xtEQ7OAG+1eDBwFkZRQGm/QaOwNgr+c80MIx9KI6IbM2rAM+4qQt0qIWQIcQ3QAItacG2dAT/aBw1pQaxAUo3JABlfMfzgIP29ZVSAM5TJMxK4Vqy/MPoRAJWyMQ9MAOiGCBaqAJ87BHFHEPMlBGL+AL2XUsiiIKWyddaINMSPZnoQSMIvRzofBZyeEIaYAGaYcIYlSMFhEOiodJvGBupNIJGuCDCFGERgUUGuA8CrEPj5AGC3AAaAAI7DgSw/COSsQLEiEKF1B2aOE7GlBHJ7MNnacGB9AAcAAkfuELundN+3AOokCApCKSFQESOYUezmiM3BAIaacGatAGU/f+FxwJA2fwD6lgBTRyEvHxEwQzM6OyD9GQLBmgARMEP/6QD9ywDajYkm4Qk53RC17wD8JwAihQA+YAGA4hLMsTIva2RZARCjohCpCgARUQCbVVHr1HD8rQBrW3BnLglH8RH/5gDiuAAnaZAqnAS0DBj7zTDd0gWzG0Gv0XHtPADtawD45Ae7X3BxqJHTuJDjVgl5J5A00UD6fBIdoQLP9QDXDQlmqQCAXZGftQBZJplyfwBVnYVtgxIesgCHGoBmygCUKlCilQmiiwAqVQD/eQUxKBXQZxcgOBHuyQCIrJfNUQmggiD1BgmyiQAlvQcyk5ED7IEE3yGDsRHvEQCMX+GYWXsBFBpFWzQALMeQJSoAo4Eg+a9EG8VEfBcRzmcXLxcAtRQAv+wJZtCQhQlg/I6Zin4Q+kUJfjOQWjEAoOwRrxARnn6R8480L7QA1jUANUeQP/cAn7h4HotZNGNQoiwJySKQJSsAWpMAvUgCP1YD0Y0Xi6QApcYANUWZoncA7toAZ4UAwjhhD+oAol0KK22QEfAAKmCQIhEKQhAAI6yqF2+QX/gA2OGF3l8AUkUKQbEAAcYKRUapo2QAq+WaPlUQ6kAAUb6gECUKXjOQJVkAojMkJaOiM2cg61AAZVYAIjMKQnMKcnAKQjYAJWAAa5EJ1pKhGypRHmQQ/0UKIOD4GPfXou+5kaWjlMAQEAOw=="
            />
            <figcaption resource="sa:bmc-res-notes-vergara-2012/F1" typeof="schema:ImageObject">
              <header><cite property="schema:alternateName">Figure 1</cite>
              </header>
              <div property="schema:caption">
                <p><strong>Distribution of SNVs, insertions, and deletions along the </strong><strong><em>C. elegans </em></strong><strong>genome for Hawaiian isolate CB4856.</strong> Segmented rings on the outside represent the six <em>C. elegans</em> chromosomes.
                  Going from outside to inside, the line plot shows SNV density (inward pointing peaks = higher density), histograms represent the density of deletions (also drawn inwards), and the heatmap depicts the density of insertions (dark red =
                  higher density) detected in the Hawaiian isolate CB4856. Note the generally higher density of SNVs towards the telomeres and the presence of chromosome-internal peaks on chromosome IV and V. Data points for this image were automatically
                  generated by CooVar using the --<em>circos</em> option. Circos was then used to generate the image. Circos configuration files necessary to create this type of image are provided with the <em>C. elegans</em> test data set at <a href="http://genome.sfu.ca/projects/coovar/"
                  class="uri">http://genome.sfu.ca/projects/coovar/</a>.</p>
              </div>
            </figcaption>
          </figure>
          <p>The second dataset contains 4,044,200 human GVs detected in an anonymous individual (HG00732-200-37-ASM) sequenced by Complete Genomics. This data set was recently made publicly available for the research community as part of a larger 69 genome
            data set [<a href="#B17" class="bibr">17</a>,<a href="#B18" class="bibr">18</a>]. HG00732-200-37-ASM variants were first extracted from the 69-sample VCF file using <em>vcf-subset</em>[<a href="#B10" class="bibr">10</a>]. We then discarded
            all but the first alternative allele and used the filtered VCF file as input to CooVar. The genomic reference sequence and the protein-coding gene set were both obtained from the Ensembl web site (release GRCh37.68, hg19). For comparison,
            we annotated the exact same VCF file with Ensembl’s Variant Effect Predictor (VEP) [<a href="#B5" class="bibr">5</a>]. VEP was run locally using the Perl script <em>variant_effect_predictor</em>.<em>pl</em> and configured to retrieve Ensembl
            data (release 68) over the internet (−−<em>host useastdb.ensembl.org</em>). The VEP output for the HG00732-200-37-ASM data set can be downloaded from the CooVar project homepage.</p>
          <p>Results of this comparison are summarized in Table <a href="#T1" class="table">1</a>. CooVar took 36 minutes to process the complete human data set and reported 4,158,840 annotated variants. VEP outputted 4,043,939 variants and finished within
            37 hours and 24 minutes. The overall increased number of variants reported by CooVar is because CooVar decomposes compound input VCF variants into sometimes multiple SNV and indel variants, which are then annotated and reported separately.
            For example, CooVar decomposes a VCF input variant with the reference allele “ATG” and the alternative allele “AC” into one SNV (T-&amp;gt;C) and one deletion variant (with G being the deleted base in this case). In contrast, VEP will annotate
            such compound variants as a single variant, resulting in sometimes ambiguous or nonspecific classification results (see examples below).</p>
          <div about="sa:bmc-res-notes-vergara-2012" rel="schema:hasPart" id="T1">
            <table resource="sa:bmc-res-notes-vergara-2012/T1" typeof="schema:Dataset">
              <caption>
                <header><cite property="schema:alternateName">Table 1</cite>
                </header>
                <div property="schema:description">
                  <p>Comparison of GVs annotated with CooVar and VEP for human individual HG00732-200-37-ASM</p>
                </div>
                <footer rel="schema:comment" about="sa:bmc-res-notes-vergara-2012/T1">
                  <div typeof="schema:Comment">
                    <div property="schema:text">
                      <p>Each GV reported by CooVar (v0.05) and VEP (v2.6) was assigned to one (and only one) of the above categories. $ CooVar: Grantham score <em>conservative</em> or <em>moderately conservative</em>; VEP: SIFT <em>benign</em>; + CooVar: Grantham
                        score <em>moderately radical</em> or <em>radical</em>; VEP: SIFT <em>deleterious.</em> * GVs assigned to more than one category due to differential impact on different transcripts. ¥ Presence of internal stop codon within the first
                        70% of ORF length, after applying all variants. § Predicted frameshift or stop gain variant within the first 70% of ORF length. Abbreviations: GV … genomic variation; ORF … open reading frame; VEP… Variant Effect Predictor; UTR…
                        untranslated region.</p>
                    </div>
                  </div>
                </footer>
              </caption>
              <colgroup>
                <col />
                <col />
                <col />
              </colgroup>
              <thead>
                <tr>
                  <th> </th>
                  <th><strong>CooVar</strong>
                  </th>
                  <th><strong>VEP</strong>
                  </th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td>Runtime
                    <hr />
                  </td>
                  <td>36m
                    <hr />
                  </td>
                  <td>37h 24m
                    <hr />
                  </td>
                </tr>
                <tr>
                  <td>Total reported GVs
                    <hr />
                  </td>
                  <td>4,158,840
                    <hr />
                  </td>
                  <td>4,043,939
                    <hr />
                  </td>
                </tr>
                <tr>
                  <td> Intronic/intergenic/UTR
                    <hr />
                  </td>
                  <td>4,133,885
                    <hr />
                  </td>
                  <td>4,019,490
                    <hr />
                  </td>
                </tr>
                <tr>
                  <td> Impacting protein-coding exon
                    <hr />
                  </td>
                  <td>24,955
                    <hr />
                  </td>
                  <td>24,449
                    <hr />
                  </td>
                </tr>
                <tr>
                  <td>  Synonymous/stop retained
                    <hr />
                  </td>
                  <td>11,585
                    <hr />
                  </td>
                  <td>11,434
                    <hr />
                  </td>
                </tr>
                <tr>
                  <td>  Missense
                    <hr />
                  </td>
                  <td>12,011
                    <hr />
                  </td>
                  <td>11,576
                    <hr />
                  </td>
                </tr>
                <tr>
                  <td>   Conservative (%)<sup>$</sup>
                    <hr />
                  </td>
                  <td>9,526 (79.3)
                    <hr />
                  </td>
                  <td>7,447 (64.3)
                    <hr />
                  </td>
                </tr>
                <tr>
                  <td>   Non-conservative (%)<sup>+</sup>
                    <hr />
                  </td>
                  <td>2,485 (20.7)
                    <hr />
                  </td>
                  <td>2,110 (18.3)
                    <hr />
                  </td>
                </tr>
                <tr>
                  <td>   Unknown consequence (%)
                    <hr />
                  </td>
                  <td>0 (0)
                    <hr />
                  </td>
                  <td>2,019 (17.4)
                    <hr />
                  </td>
                </tr>
                <tr>
                  <td>  Splice donor/acceptor
                    <hr />
                  </td>
                  <td>97
                    <hr />
                  </td>
                  <td>184
                    <hr />
                  </td>
                </tr>
                <tr>
                  <td>  Stop lost
                    <hr />
                  </td>
                  <td>47
                    <hr />
                  </td>
                  <td>46
                    <hr />
                  </td>
                </tr>
                <tr>
                  <td>  Stop gained
                    <hr />
                  </td>
                  <td>137
                    <hr />
                  </td>
                  <td>134
                    <hr />
                  </td>
                </tr>
                <tr>
                  <td>  Frameshift
                    <hr />
                  </td>
                  <td>470
                    <hr />
                  </td>
                  <td>490
                    <hr />
                  </td>
                </tr>
                <tr>
                  <td>  Inframe
                    <hr />
                  </td>
                  <td>199
                    <hr />
                  </td>
                  <td>165
                    <hr />
                  </td>
                </tr>
                <tr>
                  <td>  Other
                    <hr />
                  </td>
                  <td>0
                    <hr />
                  </td>
                  <td>31
                    <hr />
                  </td>
                </tr>
                <tr>
                  <td>  Multiple*
                    <hr />
                  </td>
                  <td>409
                    <hr />
                  </td>
                  <td>389
                    <hr />
                  </td>
                </tr>
                <tr>
                  <td>ORF disrupted</td>
                  <td>782<sup>¥</sup>
                  </td>
                  <td>871<sup>§</sup>
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
          <p>As expected, both programs classify the vast majority of variants as not impacting protein-coding exons (Table <a href="#T1" class="table">1</a>, category <em>intronic/intergenic/UTR</em>). Only 0.6% of all variants (24,955 variants by CooVar
            and 24,449 by VEP) are predicted to impact protein-coding exons in some form. To allow for a detailed comparison of annotation results, we assigned variants impacting protein-coding exons into one (and only one) of the following categories:
            variants not altering protein translation (<em>synonymous/stop retained</em>); variants altering protein translation (<em>missense</em>); variants impacting AG/GT splice site di-nucleotides (<em>splice donor/acceptor</em>); variants leading
            to stop codon loss (<em>stop lost</em>) or gain (<em>stop gained</em>); and insertions or deletions that shift (<em>frameshift</em>) or preserve the open reading frame (<em>inframe</em>). Thirty-one VEP variants could not be assigned to one
            of these categories and were classified as <em>other</em>. This includes variants that VEP nonspecifically annotated as <em>coding_sequence_variant</em>. A number of variants (409 for CooVar, 389 for VEP) could not be unambiguously assigned
            to a single category because they impact multiple transcripts differently and were classified as <em>multiple</em>.</p>
          <p>Overall, we find that numbers of GVs in each category agree well between CooVar and VEP (Table <a href="#T1" class="table">1</a>). Both programs predict ~11,500 synonymous variants and about the same number of missense variants. CooVar’s Grantham
            score classifies ~20% of missense variants as <em>moderately radical</em> or <em>radical</em>, which agrees well with the VEP SIFT classification scheme that predicts 18% of missense variants to be <em>deleterious</em>. Both programs predict about
            50 stop lost mutations and 135 stop gain mutations. Interestingly, VEP predicts 20 more frameshift variants than CooVar (490 vs. 470 variants) and 34 less inframe variants (165 vs. 199 variants). Also, the number of predicted splice site variants
            is markedly different between the two programs, with almost twice as many splice site variants predicted by VEP (184 variants) than CooVar (97 variants).</p>
          <p>To understand the nature of these differences, we performed a detailed manual analysis of GVs that were differently annotated between CooVar and VEP. In general, we find that the main source of discrepancy between CooVar and VEP is due to the
            fact that CooVar but not VEP recognized the presence of SNVs within more complex or compound VCF input variants. For example, reference and alternative allele in the VCF input variant 11:11,292,688:GGGTCAGGACGC<u>G</u>-&amp;gt;GGGTCAGGACGC<u>C</u> differ
            by only a single SNV (G-&amp;gt;C, underlined). CooVar correctly reports this variant as synonymous SNV while VEP annotates it less specifically as <em>coding_sequence_variant,</em> without information on codon impact. The different numbers
            in stop lost and stop gained variants are attributable to the same effect. For example, CooVar interprets multi-SNV variant 19:43,922,549:AGA-&amp;gt;TGC as stop lost variant while VEP annotates it as <em>coding_sequence_variant</em> and <em>3_prime_UTR_variant</em>.
            Manual inspection showed that the first of the three SNVs encoded by this input variant (A-&amp;gt;T) indeed changes the stop codon of transcript ENST00000253435 from TAA to TAT, suggesting that the CooVar prediction is correct.</p>
          <p>The much larger number of splice site variants predicted by VEP is also explained by the higher resolution with which CooVar decomposes complex VCF input variants. For example, variant 11:117,303,853:CCCAGT-&amp;gt;CCCAGC is annotated as splice
            donor variant by VEP but not CooVar, which reports it as synonymous SNV. Manual inspection showed that the coordinates of this variant (117,303,853–117,303,858) indeed overlap with a donor splice site of transcript ENST00000527706, but the
            actual SNV encoded by this variant (T-&amp;gt;C) is in fact synonymous. Thus, in this case, a simple coordinate overlap analysis as seems to be performed by VEP produces an incorrect result. Other examples of this type include variant 7:101,194,424:CGTAA-&amp;gt;TGTAA
            (CooVar: synonymous), 5:159,835,654:TACCA-&amp;gt;TACCG (CooVar: missense), or 19:16,612,363:GTG-&amp;gt;GTA (CooVar: silent).</p>
          <p>Complex input variants also account for discrepancies observed between indel annotations. We randomly picked and examined 10 of the 34 inframe variants predicted by CooVar but not VEP. For 9 out of these, we find that they are genuine inframe
            indels that VEP classified as missense (for example 10:126,715,151:TGCAGAGGAGC-&amp;gt;TGCGGAGGAGCCGCAGGCTGGGGCTGCAGGGC or 12:53,045,625:CT-&amp;gt;CCGCTGCCGCCTCCAAAGCC; note the length difference is a multiple of 3 in both cases). Why VEP
            classifies these variants as missense variants was not obvious to us. The remaining variant of these ten variants was actually classified as inframe variant by VEP but assigned to category <em>multiple</em> by our classification scheme because
            VEP predicts it as both inframe and stop gained variant.</p>
          <p>Another main source of discrepancy in indel classification arose from so called “boundary indels”. We refer to boundary indels as indels that fall right next to the start or end of coding exons, thus leaving some uncertainty about the exact
            impact of these variants on the protein-coding transcript. Variant 7:142,494,013 is an example of an insertion where the exact placement of the inserted sequence is ambiguous, resulting in a predicted frameshift insertion by CooVar but in
            a predicted <em>coding_sequence_variant</em> and <em>5_prime_UTR_variant</em> by VEP. Most of the frameshift variants predicted by VEP but not CooVar represent boundary indels. Representative examples include 11:111,853,106:G-&amp;gt;GC (1-bp
            insertion right before coding exon), 16:76,311,602:G-&amp;gt;GT (1-bp insertion right after coding exon), 16:31,770,696:GA-&amp;gt;GAA (1-bp insertion into start codon), and 17:39,254,335:AT-&amp;gt;ATT (1-bp insertion before start codon).
            We manually inspected all 20 frameshift variants predicted by VEP but not CooVar and confirm that CooVar predictions appear to be correct, <em>i.e.</em> these variants are likely not causing frameshift mutations in affected transcripts.</p>
          <p>We were also interested in the number of ORFs that were predicted to be disrupted by both CooVar and VEP. For this particular comparison, we defined a CooVar ORF as being disrupted if an internal stop codon occurred within the first 70% of the
            ORF’s length after applying all GVs to a transcript. CooVar provides the position of the first internal stop codon as part of its output. VEP does not provide ORF status information in its output, so we defined a VEP ORF as being disrupted
            if VEP predicted at least one frameshift or stop gain variant within the first 70% of the ORF’s length. Using these criteria, we find that CooVar predicts 782 ORFs to be disrupted while VEP predicts 871 ORFs as disrupted (Table <a href="#T1"
            class="table">1</a>). We inspected about half (48) of the transcripts that had assigned a different ORF status by the two programs and found that most of them (20 ORFs, <em>e.g.</em> transcript ENST00000376343) carry a frame-shifting indel that
            does not introduce an internal stop codon albeit it changes the translated protein sequence downstream. Thus, although for these transcripts a significant portion of the ORF (&amp;gt;30%) is changed in terms of its protein sequence, the length
            of the ORF remains intact. Seventeen of the 48 inspected transcripts (<em>e.g.</em> ENST00000222270) had a frameshift predicted by VEP but not CooVar due to boundary indels as discussed above. Five of the 48 transcripts had already internal
            stop codons in the reference sequence and hence were not annotated as disrupted by CooVar.</p>
          <p>Most importantly, the remaining six ORFs predicted to be disrupted by VEP but not CooVar carry neighboring indels that cancel each others effect, restoring the open reading frame. Figure <a href="#F2" class="fig">2A</a> shows one such example
            affecting Ensembl transcript ENST00000253255. This transcript carries a 1bp insertion at position 22:46,658,224 and a nearby 1bp deletion at position 22:46,658,220. When evaluated independently, the impact of the insertion and the deletion
            on this transcript is a frameshift mutation, disrupting the ORF. But when evaluating the joint effect that these GVs have on the transcript it results in a preserved ORF, as reported by CooVar. A similar issue arises from co-occurring SNVs.
            In the example shown in Figure <a href="#F2" class="fig">2B</a>, VEP classifies SNV 10:27,702,726:G-&amp;gt;A as synonymous because it changes codon CTG on the reverse strand (coding for leucine) to TTG (also coding for leucine). However,
            CooVar considers the combined effect of this SNV and a neighboring SNV (10:27,702,725:A-&amp;gt;G) that affects the same codon. When evaluated together, 10:27702726:G-&amp;gt;A is recognized by CooVar as missense SNV that changes the codon
            from CTG to TCG, which codes for amino acid serine. These two examples illustrate the importance of assessing the impact of co-occurring GVs together to correctly judge their functional impact [<a href="#B9" class="bibr">9</a>].</p>
          <figure
          about="sa:bmc-res-notes-vergara-2012" rel="schema:hasPart" id="F2">
            <img about="sa:bmc-res-notes-vergara-2012/F2" src="data:image/gif;base64,R0lGODlhYAAWAPUAAHR0dOAmJucnJ/8nJ6V8fOBfX/xQUPVqavZ9fV6nXnyPfAzNzQ3NzQ7OzhrY2BrZ2TbV1Wne3v+5udTf1NjY2N7e3t/f3//Ozu/a2uDg4OHh4eLi4uPj4+Tk5Ofn5+jo6Onp6erq6uvr6+zs7O3t7e7u7u/v7/Dw8PHx8fLy8vPz8/T09PX19fb29vf39/j4+Pn5+fr6+vv7+/z8/P39/f7+/v///wAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACH5BAEAADcALAAAAABgABYAAAbPwNtm0xGJKsfX5kVJiVivFCtlo5psourLNDW9suDqxYCI2q6ixKSK3VLf2HhWmkLL2aItNHuF3/6AgYKDhIWDBIQKhouMjY6PkJGSk5SVlpeWD5qbnJ2en6ChoqOkpZyYqKmqq6ytrq+wsbKztLW2t7i5uru8vb6/wMHCugDDtCkpHVs2HTYbNhVVXczO1c0dLCIpZs/PV0tSZgUDAgMY1VwiJnVn6O3a69RXVx0SAQcm697uV8b+/wADTorQoKDBgwgTKlzIsKHDhxANQggCADs="
            />
            <figcaption resource="sa:bmc-res-notes-vergara-2012/F2" typeof="schema:ImageObject">
              <header><cite property="schema:alternateName">Figure 2</cite>
              </header>
              <div property="schema:caption">
                <p><strong>GVs affecting the same protein-coding transcript must be assessed together to correctly predict their functional impact.</strong> Panel <strong>A</strong> shows an example where two neighboring frameshift indels (1-bp insertion and
                  1-bp deletion, indicated by arrows) cancel each others effect, restoring the original ORF. Panel <strong>B</strong> shows an example where an otherwise synonymous SNV (G-&amp;gt;A, indicated by arrow) causes a missense mutation due to
                  the effect of a neighboring SNV (A-&amp;gt;G). In both panels, the first three rows show the reference nucleotide sequence on the forward strand, the reference nucleotide sequence from the reverse strand, and the reference protein sequence
                  translation from the annotated ORF. Note that both ORFs are encoded on the reverse strand, so sequences must be read from right to left. The track below shows the variant sequence detected in human individual HG00732-200-37-ASM, with
                  critical GVs highlighted by arrows. The blue horizontal bar represents the Ensembl protein-coding transcript spanning this genomic region.</p>
              </div>
            </figcaption>
            </figure>
            <p>We conclude that CooVar is a fast and light-weight alternative to currently existing variant annotation tools that is particularly useful for non-model organisms. CooVar produces very similar results as other popular tools, but, under certain
              circumstances, generates biologically more accurate annotations by considering the combined effect of co-occurring GVs on protein-coding transcripts.</p>
        </section>
      </section>
      <section>
        <h2>Availability and requirements</h2>
        <p><strong>Project name</strong>: CooVar: Co-occurring Variant Analyzer</p>
        <p><strong>Project home page</strong>: <a href="http://genome.sfu.ca/projects/coovar" class="uri">http://genome.sfu.ca/projects/coovar</a>
        </p>
        <p><strong>Operating system(s)</strong>: Windows, Linux, Mac OS-X</p>
        <p><strong>Programming language</strong>: Perl 5.8.8</p>
        <p><strong>Other requirements</strong>: The following Perl modules are required by CooVar and need to be installed: Cwd, Getopt::Long, POSIX, File::Basename, List::Util, Bio::DB::Fasta, Bio::Seq, Bio::SeqUtils, Bio::SeqIO, Set::IntervalTree, Set::IntSpan</p>
        <p><strong>License</strong>: GNU GPL</p>
        <p><strong>Any restrictions to use by non-academics</strong>: none</p>
        <p>The latest version of the program can be obtained from the project webpage. CooVar version 0.05 is included as online supplementary material (Additional file <a href="#S1" class="supplementary-material">1</a>).</p>
      </section>
      <section>
        <h2>Abbreviations</h2>
        <p>GV: Genomic Variation; GFF: Generic Feature Format; GTF: Gene Transfer Format; SNV: Single Nucleotide Variant; Indel: Insertion or deletion; GVF: Genomic Variant Format; ORF: Open Reading Frame; VCF: Variant Call Format.</p>
      </section>
      <section>
        <h2>Competing interests</h2>
        <p>The authors declare that they have no competing interest.</p>
      </section>
      <section>
        <h2>Authors’ contribution</h2>
        <p>IAV and NC conceived the study. IAV developed the method and original program. CF co-developed and tested the program. IAV, CF, and NC wrote the manuscript. All authors read and approved the final manuscript.</p>
      </section>
      <section>
        <h2>Authors’ information</h2>
        <p>Ismael A Vergara and Christian Frech were joint first author.</p>
      </section>
      <section>
        <h2>Supplementary Material</h2>
        <aside about="sa:bmc-res-notes-vergara-2012" rel="schema:hasPart" id="S1">
          <div resource="sa:bmc-res-notes-vergara-2012/S1" typeof="schema:CreativeWork">
            <header>
              <h3 property="schema:headline">Additional file 1</h3>
            </header>
            <div property="schema:description">
              <p>CooVar program tarball (version 0.05), including README and test scripts.</p>
            </div>
          </div>
        </aside>
      </section>
    </section>
    <section typeof="doco:BackMatter">
      <section typeof="deo:Acknowledgements">
        <h3>Acknowledgements</h3>
        <p>The authors would like to acknowledge Tammy Wong, Jeffrey Chu, Shanshan Zou, Jiarui Li and all other lab members for insightful discussion and for testing the program. We thank Duncan Napier for IT support. This study is supported by a Discovery
          Grant from the Natural Sciences and Engineering Research Council of Canada (NSERC) to NC. NC is a Michael Smith Foundation for Health Research (MSFHR) Scholar and a Canadian Institutes of Health Research (CIHR) New Investigator. IAV was supported
          by an MBB Alumni Graduate Scholarship. CF received SFU graduate fellowships and was supported by BC Pacific Century, Weyerhaeuser, and Sulzer Pumps graduate scholarships.</p>
      </section>
      <section typeof="doco:Bibliography">
        <ul typeof="doco:BibliographicReferenceList">
          <li id="B1" rel="schema:citation" about="sa:bmc-res-notes-vergara-2012">
            <div typeof="schema:ScholarlyArticle"><span property="schema:author" typeof="schema:Person"><span property="schema:familyName">MacArthur</span>  <span property="schema:givenName">DG</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Tyler-Smith</span> 
              <span
              property="schema:givenName">C</span>
                </span> <span property="schema:datePublished" content="2009-12-31T19:00:00.000Z">(2009)</span>  <span property="schema:headline">Loss-of-function variants in the genomes of healthy humans</span>  <span property="schema:isPartOf" typeof="schema:PublicationIssue"><span property="schema:isPartOf" typeof="schema:PublicationVolume"><span property="schema:isPartOf" typeof="schema:Periodical"><span property="schema:name">Hum Mol Genet</span></span>
                <span
                property="schema:volumeNumber">19</span>
                  </span>(<span property="bibo:issue">R2</span>)</span>:<span property="schema:pageStart">R125</span>-<span property="schema:pageEnd">R130</span>.</div>
          </li>
          <li id="B2" rel="schema:citation" about="sa:bmc-res-notes-vergara-2012">
            <div typeof="schema:ScholarlyArticle"><span property="schema:author" typeof="schema:Person"><span property="schema:familyName">Stankiewicz</span>  <span property="schema:givenName">P</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Lupski</span> 
              <span
              property="schema:givenName">JR</span>
                </span> <span property="schema:datePublished" content="2009-12-31T19:00:00.000Z">(2009)</span>  <span property="schema:headline">Structural variation in the human genome and its role in disease</span>  <span property="schema:isPartOf" typeof="schema:PublicationVolume"><span property="schema:isPartOf" typeof="schema:Periodical"><span property="schema:name">Annu Rev Med</span></span>
                <span
                property="schema:volumeNumber">61</span>
                  </span>:<span property="schema:pageStart">437</span>-<span property="schema:pageEnd">455</span>.</div>
          </li>
          <li id="B3" rel="schema:citation" about="sa:bmc-res-notes-vergara-2012">
            <div typeof="schema:ScholarlyArticle"><span property="schema:author" typeof="schema:Person"><span property="schema:familyName">Shendure</span>  <span property="schema:givenName">J</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Ji</span> 
              <span
              property="schema:givenName">H</span>
                </span> <span property="schema:datePublished" content="2007-12-31T19:00:00.000Z">(2007)</span>  <span property="schema:headline">Next-generation DNA sequencing</span>  <span property="schema:isPartOf" typeof="schema:PublicationIssue"><span property="schema:isPartOf" typeof="schema:PublicationVolume"><span property="schema:isPartOf" typeof="schema:Periodical"><span property="schema:name">Nat Biotechnol</span></span>
                <span
                property="schema:volumeNumber">26</span>
                  </span>(<span property="bibo:issue">10</span>)</span>:<span property="schema:pageStart">1135</span>-<span property="schema:pageEnd">1145</span>.</div>
          </li>
          <li id="B4" rel="schema:citation" about="sa:bmc-res-notes-vergara-2012">
            <div typeof="schema:ScholarlyArticle"><span property="schema:author" typeof="schema:Person"><span property="schema:familyName">Medvedev</span>  <span property="schema:givenName">P</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Stanciu</span> 
              <span
              property="schema:givenName">M</span>
                </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Brudno</span>  <span property="schema:givenName">M</span></span> <span property="schema:datePublished" content="2008-12-31T19:00:00.000Z">(2008)</span> 
                <span
                property="schema:headline">Computational methods for discovering structural variation with next-generation sequencing</span> <span property="schema:isPartOf" typeof="schema:PublicationIssue"><span property="schema:isPartOf" typeof="schema:PublicationVolume"><span property="schema:isPartOf" typeof="schema:Periodical"><span property="schema:name">Nat Methods</span></span>
                  <span
                  property="schema:volumeNumber">6</span>
                    </span>(<span property="bibo:issue">11 Suppl</span>)</span>:<span property="schema:pageStart">S13</span>-<span property="schema:pageEnd">S20</span>.</div>
          </li>
          <li id="B5" rel="schema:citation" about="sa:bmc-res-notes-vergara-2012">
            <div typeof="schema:ScholarlyArticle"><span property="schema:author" typeof="schema:Person"><span property="schema:familyName">McLaren</span>  <span property="schema:givenName">W</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Pritchard</span> 
              <span
              property="schema:givenName">B</span>
                </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Rios</span>  <span property="schema:givenName">D</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Chen</span> 
                <span
                property="schema:givenName">Y</span>
                  </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Flicek</span>  <span property="schema:givenName">P</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Cunningham</span> 
                  <span
                  property="schema:givenName">F</span>
                    </span> <span property="schema:datePublished" content="2009-12-31T19:00:00.000Z">(2009)</span>  <span property="schema:headline">Deriving the consequences of genomic variants with the Ensembl API and SNP Effect Predictor</span>  <span property="schema:isPartOf"
                    typeof="schema:PublicationIssue"><span property="schema:isPartOf" typeof="schema:PublicationVolume"><span property="schema:isPartOf" typeof="schema:Periodical"><span property="schema:name">Bioinformatics</span></span> <span property="schema:volumeNumber">26</span></span>(
                    <span
                    property="bibo:issue">16</span>)</span>:<span property="schema:pageStart">2069</span>-<span property="schema:pageEnd">2070</span>.</div>
          </li>
          <li id="B6" rel="schema:citation" about="sa:bmc-res-notes-vergara-2012">
            <div typeof="schema:ScholarlyArticle"><span property="schema:author" typeof="schema:Person"><span property="schema:familyName">McKenna</span>  <span property="schema:givenName">A</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Hanna</span> 
              <span
              property="schema:givenName">M</span>
                </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Banks</span>  <span property="schema:givenName">E</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Sivachenko</span> 
                <span
                property="schema:givenName">A</span>
                  </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Cibulskis</span>  <span property="schema:givenName">K</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Kernytsky</span> 
                  <span
                  property="schema:givenName">A</span>
                    </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Garimella</span>  <span property="schema:givenName">K</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Altshuler</span> 
                    <span
                    property="schema:givenName">D</span>
                      </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Gabriel</span>  <span property="schema:givenName">S</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Daly</span> 
                      <span
                      property="schema:givenName">M</span>
                        </span> <em property="pkg:unnamedContributors" content="true">et al.</em>  <span property="schema:datePublished" content="2009-12-31T19:00:00.000Z">(2009)</span>  <span property="schema:headline">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</span> 
                        <span
                        property="schema:isPartOf" typeof="schema:PublicationIssue"><span property="schema:isPartOf" typeof="schema:PublicationVolume"><span property="schema:isPartOf" typeof="schema:Periodical"><span property="schema:name">Genome Res</span></span> <span property="schema:volumeNumber">20</span></span>(
                          <span
                          property="bibo:issue">9</span>)</span>:<span property="schema:pageStart">1297</span>-<span property="schema:pageEnd">1303</span>.</div>
          </li>
          <li id="B7" rel="schema:citation" about="sa:bmc-res-notes-vergara-2012">
            <div typeof="schema:ScholarlyArticle"><span property="schema:author" typeof="schema:Person"><span property="schema:familyName">Ge</span>  <span property="schema:givenName">D</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Ruzzo</span> 
              <span
              property="schema:givenName">EK</span>
                </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Shianna</span>  <span property="schema:givenName">KV</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">He</span> 
                <span
                property="schema:givenName">M</span>
                  </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Pelak</span>  <span property="schema:givenName">K</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Heinzen</span> 
                  <span
                  property="schema:givenName">EL</span>
                    </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Need</span>  <span property="schema:givenName">AC</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Cirulli</span> 
                    <span
                    property="schema:givenName">ET</span>
                      </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Maia</span>  <span property="schema:givenName">JM</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Dickson</span> 
                      <span
                      property="schema:givenName">SP</span>
                        </span> <em property="pkg:unnamedContributors" content="true">et al.</em>  <span property="schema:datePublished" content="2010-12-31T19:00:00.000Z">(2010)</span>  <span property="schema:headline">SVA: software for annotating and visualizing sequenced human genomes</span> 
                        <span
                        property="schema:isPartOf" typeof="schema:PublicationIssue"><span property="schema:isPartOf" typeof="schema:PublicationVolume"><span property="schema:isPartOf" typeof="schema:Periodical"><span property="schema:name">Bioinformatics</span></span> <span property="schema:volumeNumber">27</span></span>(
                          <span
                          property="bibo:issue">14</span>)</span>:<span property="schema:pageStart">1998</span>-<span property="schema:pageEnd">2000</span>.</div>
          </li>
          <li id="B8" rel="schema:citation" about="sa:bmc-res-notes-vergara-2012">
            <div typeof="schema:ScholarlyArticle"><span property="schema:author" typeof="schema:Person"><span property="schema:familyName">Wang</span>  <span property="schema:givenName">K</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Li</span> 
              <span
              property="schema:givenName">M</span>
                </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Hakonarson</span>  <span property="schema:givenName">H</span></span> <span property="schema:datePublished" content="2009-12-31T19:00:00.000Z">(2009)</span> 
                <span
                property="schema:headline">ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data</span> <span property="schema:isPartOf" typeof="schema:PublicationIssue"><span property="schema:isPartOf" typeof="schema:PublicationVolume"><span property="schema:isPartOf" typeof="schema:Periodical"><span property="schema:name">Nucleic Acids Res</span></span>
                  <span
                  property="schema:volumeNumber">38</span>
                    </span>(<span property="bibo:issue">16</span>)</span>:<span property="schema:pageStart">e164</span>.</div>
          </li>
          <li id="B9" rel="schema:citation" about="sa:bmc-res-notes-vergara-2012">
            <div typeof="schema:ScholarlyArticle"><span property="schema:author" typeof="schema:Person"><span property="schema:familyName">MacArthur</span>  <span property="schema:givenName">DG</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Balasubramanian</span> 
              <span
              property="schema:givenName">S</span>
                </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Frankish</span>  <span property="schema:givenName">A</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Huang</span> 
                <span
                property="schema:givenName">N</span>
                  </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Morris</span>  <span property="schema:givenName">J</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Walter</span> 
                  <span
                  property="schema:givenName">K</span>
                    </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Jostins</span>  <span property="schema:givenName">L</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Habegger</span> 
                    <span
                    property="schema:givenName">L</span>
                      </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Pickrell</span>  <span property="schema:givenName">JK</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Montgomery</span> 
                      <span
                      property="schema:givenName">SB</span>
                        </span> <em property="pkg:unnamedContributors" content="true">et al.</em>  <span property="schema:datePublished" content="2011-12-31T19:00:00.000Z">(2011)</span>  <span property="schema:headline">A systematic survey of loss-of-function variants in human protein-coding genes</span> 
                        <span
                        property="schema:isPartOf" typeof="schema:PublicationIssue"><span property="schema:isPartOf" typeof="schema:PublicationVolume"><span property="schema:isPartOf" typeof="schema:Periodical"><span property="schema:name">Science</span></span> <span property="schema:volumeNumber">335</span></span>(
                          <span
                          property="bibo:issue">6070</span>)</span>:<span property="schema:pageStart">823</span>-<span property="schema:pageEnd">828</span>.</div>
          </li>
          <li id="B10" rel="schema:citation" about="sa:bmc-res-notes-vergara-2012">
            <div typeof="schema:ScholarlyArticle"><span property="schema:author" typeof="schema:Person"><span property="schema:familyName">Danecek</span>  <span property="schema:givenName">P</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Auton</span> 
              <span
              property="schema:givenName">A</span>
                </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Abecasis</span>  <span property="schema:givenName">G</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Albers</span> 
                <span
                property="schema:givenName">CA</span>
                  </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Banks</span>  <span property="schema:givenName">E</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">DePristo</span> 
                  <span
                  property="schema:givenName">MA</span>
                    </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Handsaker</span>  <span property="schema:givenName">RE</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Lunter</span> 
                    <span
                    property="schema:givenName">G</span>
                      </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Marth</span>  <span property="schema:givenName">GT</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Sherry</span> 
                      <span
                      property="schema:givenName">ST</span>
                        </span> <em property="pkg:unnamedContributors" content="true">et al.</em>  <span property="schema:datePublished" content="2010-12-31T19:00:00.000Z">(2010)</span>  <span property="schema:headline">The variant call format and VCFtools</span> 
                        <span
                        property="schema:isPartOf" typeof="schema:PublicationIssue"><span property="schema:isPartOf" typeof="schema:PublicationVolume"><span property="schema:isPartOf" typeof="schema:Periodical"><span property="schema:name">Bioinformatics</span></span> <span property="schema:volumeNumber">27</span></span>(
                          <span
                          property="bibo:issue">15</span>)</span>:<span property="schema:pageStart">2156</span>-<span property="schema:pageEnd">2158</span>.</div>
          </li>
          <li id="B11" rel="schema:citation" about="sa:bmc-res-notes-vergara-2012">
            <div typeof="schema:ScholarlyArticle"><span property="schema:author" typeof="schema:Person"><span property="schema:familyName">Reese</span>  <span property="schema:givenName">MG</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Moore</span> 
              <span
              property="schema:givenName">B</span>
                </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Batchelor</span>  <span property="schema:givenName">C</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Salas</span> 
                <span
                property="schema:givenName">F</span>
                  </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Cunningham</span>  <span property="schema:givenName">F</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Marth</span> 
                  <span
                  property="schema:givenName">GT</span>
                    </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Stein</span>  <span property="schema:givenName">L</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Flicek</span> 
                    <span
                    property="schema:givenName">P</span>
                      </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Yandell</span>  <span property="schema:givenName">M</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Eilbeck</span> 
                      <span
                      property="schema:givenName">K</span>
                        </span> <span property="schema:datePublished" content="2009-12-31T19:00:00.000Z">(2009)</span>  <span property="schema:headline">A standard variation file format for human genome sequences</span>  <span property="schema:isPartOf" typeof="schema:PublicationIssue"><span property="schema:isPartOf" typeof="schema:PublicationVolume"><span property="schema:isPartOf" typeof="schema:Periodical"><span property="schema:name">Genome Biol</span></span>
                        <span
                        property="schema:volumeNumber">11</span>
                          </span>(<span property="bibo:issue">8</span>)</span>:<span property="schema:pageStart">R88</span>.</div>
          </li>
          <li id="B12" rel="schema:citation" about="sa:bmc-res-notes-vergara-2012">
            <div typeof="schema:ScholarlyArticle"><span property="schema:author" typeof="schema:Person"><span property="schema:familyName">Eilbeck</span>  <span property="schema:givenName">K</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Lewis</span> 
              <span
              property="schema:givenName">SE</span>
                </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Mungall</span>  <span property="schema:givenName">CJ</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Yandell</span> 
                <span
                property="schema:givenName">M</span>
                  </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Stein</span>  <span property="schema:givenName">L</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Durbin</span> 
                  <span
                  property="schema:givenName">R</span>
                    </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Ashburner</span>  <span property="schema:givenName">M</span></span> <span property="schema:datePublished" content="2004-12-31T19:00:00.000Z">(2004)</span> 
                    <span
                    property="schema:headline">The Sequence Ontology: a tool for the unification of genome annotations</span> <span property="schema:isPartOf" typeof="schema:PublicationIssue"><span property="schema:isPartOf" typeof="schema:PublicationVolume"><span property="schema:isPartOf" typeof="schema:Periodical"><span property="schema:name">Genome Biol</span></span>
                      <span
                      property="schema:volumeNumber">6</span>
                        </span>(<span property="bibo:issue">5</span>)</span>:<span property="schema:pageStart">R44</span>.</div>
          </li>
          <li id="B13" rel="schema:citation" about="sa:bmc-res-notes-vergara-2012">
            <div typeof="schema:ScholarlyArticle"><span property="schema:author" typeof="schema:Person"><span property="schema:familyName">Grantham</span>  <span property="schema:givenName">R</span></span> <span property="schema:datePublished" content="1973-12-31T19:00:00.000Z">(1973)</span> 
              <span
              property="schema:headline">Amino acid difference formula to help explain protein evolution</span> <span property="schema:isPartOf" typeof="schema:PublicationIssue"><span property="schema:isPartOf" typeof="schema:PublicationVolume"><span property="schema:isPartOf" typeof="schema:Periodical"><span property="schema:name">Science</span></span>
                <span
                property="schema:volumeNumber">185</span>
                  </span>(<span property="bibo:issue">4154</span>)</span>:<span property="schema:pageStart">862</span>-<span property="schema:pageEnd">864</span>.</div>
          </li>
          <li id="B14" rel="schema:citation" about="sa:bmc-res-notes-vergara-2012">
            <div typeof="schema:ScholarlyArticle"><span property="schema:author" typeof="schema:Person"><span property="schema:familyName">Li</span>  <span property="schema:givenName">WH</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Wu</span> 
              <span
              property="schema:givenName">CI</span>
                </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Luo</span>  <span property="schema:givenName">CC</span></span> <span property="schema:datePublished" content="1983-12-31T19:00:00.000Z">(1983)</span> 
                <span
                property="schema:headline">Nonrandomness of point mutation as reflected in nucleotide substitutions in pseudogenes and its evolutionary implications</span> <span property="schema:isPartOf" typeof="schema:PublicationIssue"><span property="schema:isPartOf" typeof="schema:PublicationVolume"><span property="schema:isPartOf" typeof="schema:Periodical"><span property="schema:name">J Mol Evol</span></span>
                  <span
                  property="schema:volumeNumber">21</span>
                    </span>(<span property="bibo:issue">1</span>)</span>:<span property="schema:pageStart">58</span>-<span property="schema:pageEnd">71</span>.</div>
          </li>
          <li id="B15" rel="schema:citation" about="sa:bmc-res-notes-vergara-2012">
            <div typeof="schema:ScholarlyArticle"><span property="schema:author" typeof="schema:Person"><span property="schema:familyName">Krzywinski</span>  <span property="schema:givenName">M</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Schein</span> 
              <span
              property="schema:givenName">J</span>
                </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Birol</span>  <span property="schema:givenName">I</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Connors</span> 
                <span
                property="schema:givenName">J</span>
                  </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Gascoyne</span>  <span property="schema:givenName">R</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Horsman</span> 
                  <span
                  property="schema:givenName">D</span>
                    </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Jones</span>  <span property="schema:givenName">SJ</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Marra</span> 
                    <span
                    property="schema:givenName">MA</span>
                      </span> <span property="schema:datePublished" content="2008-12-31T19:00:00.000Z">(2008)</span>  <span property="schema:headline">Circos: an information aesthetic for comparative genomics</span>  <span property="schema:isPartOf" typeof="schema:PublicationIssue"><span property="schema:isPartOf" typeof="schema:PublicationVolume"><span property="schema:isPartOf" typeof="schema:Periodical"><span property="schema:name">Genome Res</span></span>
                      <span
                      property="schema:volumeNumber">19</span>
                        </span>(<span property="bibo:issue">9</span>)</span>:<span property="schema:pageStart">1639</span>-<span property="schema:pageEnd">1645</span>.</div>
          </li>
          <li id="B16" rel="schema:citation" about="sa:bmc-res-notes-vergara-2012">
            <div typeof="schema:ScholarlyArticle"><span property="schema:author" typeof="schema:Person"><span property="schema:familyName">Harris</span>  <span property="schema:givenName">TW</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Antoshechkin</span> 
              <span
              property="schema:givenName">I</span>
                </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Bieri</span>  <span property="schema:givenName">T</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Blasiar</span> 
                <span
                property="schema:givenName">D</span>
                  </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Chan</span>  <span property="schema:givenName">J</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Chen</span> 
                  <span
                  property="schema:givenName">WJ</span>
                    </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">De La Cruz</span>  <span property="schema:givenName">N</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Davis</span> 
                    <span
                    property="schema:givenName">P</span>
                      </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Duesbury</span>  <span property="schema:givenName">M</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Fang</span> 
                      <span
                      property="schema:givenName">R</span>
                        </span> <em property="pkg:unnamedContributors" content="true">et al.</em>  <span property="schema:datePublished" content="2009-12-31T19:00:00.000Z">(2009)</span>  <span property="schema:headline">WormBase: a comprehensive resource for nematode research</span> 
                        <span
                        property="schema:isPartOf" typeof="schema:PublicationIssue"><span property="schema:isPartOf" typeof="schema:PublicationVolume"><span property="schema:isPartOf" typeof="schema:Periodical"><span property="schema:name">Nucleic Acids Res</span></span> <span property="schema:volumeNumber">38</span></span>(
                          <span
                          property="bibo:issue">Database issue</span>)</span>:<span property="schema:pageStart">D463</span>-<span property="schema:pageEnd">D467</span>.</div>
          </li>
          <li id="B17" rel="schema:citation" about="sa:bmc-res-notes-vergara-2012">
            <div typeof="schema:ScholarlyArticle"><span property="schema:author" typeof="schema:Person"><span property="schema:familyName">Drmanac</span>  <span property="schema:givenName">R</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Sparks</span> 
              <span
              property="schema:givenName">AB</span>
                </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Callow</span>  <span property="schema:givenName">MJ</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Halpern</span> 
                <span
                property="schema:givenName">AL</span>
                  </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Burns</span>  <span property="schema:givenName">NL</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Kermani</span> 
                  <span
                  property="schema:givenName">BG</span>
                    </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Carnevali</span>  <span property="schema:givenName">P</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Nazarenko</span> 
                    <span
                    property="schema:givenName">I</span>
                      </span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Nilsen</span>  <span property="schema:givenName">GB</span></span>, <span property="schema:contributor" typeof="schema:Person"><span property="schema:familyName">Yeung</span> 
                      <span
                      property="schema:givenName">G</span>
                        </span> <em property="pkg:unnamedContributors" content="true">et al.</em>  <span property="schema:datePublished" content="2009-12-31T19:00:00.000Z">(2009)</span>  <span property="schema:headline">Human genome sequencing using unchained base reads on self-assembling DNA nanoarrays</span> 
                        <span
                        property="schema:isPartOf" typeof="schema:PublicationIssue"><span property="schema:isPartOf" typeof="schema:PublicationVolume"><span property="schema:isPartOf" typeof="schema:Periodical"><span property="schema:name">Science</span></span> <span property="schema:volumeNumber">327</span></span>(
                          <span
                          property="bibo:issue">5961</span>)</span>:<span property="schema:pageStart">78</span>-<span property="schema:pageEnd">81</span>.</div>
          </li>
          <li id="B18" rel="schema:citation" about="sa:bmc-res-notes-vergara-2012">
            <div typeof="schema:CreativeWork"><span property="schema:headline">Complete Genomics 69 Genomes Data</span> . ftp://ftp2.completegenomics.com/Multigenome_summaries/Complete_Public_Genomes_69genomes_B37_mkvcf.vcf.bz2</div>
          </li>
        </ul>
      </section>
    </section>
  </article>
</body>

</html>